daisy_workflows/image_build/enterprise_linux/almalinux_8.wf.json

```diff
@@ -0,0 +1,59 @@
+{
+  "Name": "build-almalinux-8",
```

Comment:
is AlmaLinux has a version number 8 like centos?

Reply:
Yup- its the same versioning scheme - and it will follow RHEL just like CentOS.